action_plugins/insights_config.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.plugins.action import ActionBase
+
+
+class ActionModule(ActionBase):
+
+    def run(self, tmp=None, task_vars=None):
+
+        result = super(ActionModule, self).run(tmp, task_vars)
+
+        insights_name = self._task.args.get('insights_name')
```

Comment:
Any chance we can make this fully optional?  Where it defaults to the modern client package name (and directory)?

Reply:
I'm actually doing that right now :D